src/scenes/Activities/scenes/EventHistory.js

```diff
@@ -229,186 +111,81 @@ export default class EventHistory extends Component {
     ];
 
     return (
-      <TableHead>
-        <TableRow>
-          {headerCols.map((column) => column.sortable ? this.getSortableCell(column) : this.getNonSortableCell(column))}
-        </TableRow>
-      </TableHead>
+      <Grid container spacing={0}>
+        {transactions.length ? (
```

Comment:
we can use `displayedTxs.length` here right? Then we don't need to destructure `transactions` above